1.12 Terraform documentation

anithamuthu79 opened the pull request:
https://jira.mesosphere.com/browse/DCOS-43624
https://docs.google.com/document/d/1Z25jSs1KZ14XEHeO5823oRHwtt17YJLW6sODtUygC6s/edit
https://www.mindmeister.com/1158553831

## Urgency
- [x] Blocker <!-- Ping @pavisandhu for review -->
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

anithamuthu79 commented:
@yishan-lin  and @bernadinm please review and share feedback on this PR.

yishan-lin commented:
Hey Anitha, we are not going to have a Troubleshooting & FAQ guide in time for 1.12 GA.  :) Please remove for now.

anithamuthu79 commented:
@yishan-lin I've removed "trouble-shooting and faq" folder.

anithamuthu79 commented:
@yishan-lin I've addressed your comments and have asked @niartenyaw to replace 
`You can create a DC/OS cluster on AWS using Terraform` with `You can create a DC/OS cluster on AWS using an Universal Installer` on AWS, Azure, and GCP pages under Mesosphere-supported installation methods on dev PRs.

yishan-lin commented:
Hey @anithamuthu79 - I don't see the updated content?  Apologies if I'm missing something.

yishan-lin commented on pages/1.12/installing/evaluation/index.md:
```diff
@@ -3,22 +3,22 @@ layout: layout.pug
+Terraform is an open source infrastructure automation tool which uses templates to manage infrastructure for multiple public cloud providers, service providers, and on-premises solutions. Terraform creates your infrastructure, configures resources, and manages communication between agents. The purpose of this tool is to automate most of the manual efforts of managing and maintaining distributed s ...
 
-The primary goals of using Terraform are as follows:
+The primary goals of using Universal Installers are as follows:
```
The primary goals of using "The Universal Installer" (there is only one, which is our tool)

yishan-lin commented on pages/1.12/installing/evaluation/index.md:
```diff
@@ -3,22 +3,22 @@ layout: layout.pug
+excerpt: Installing DC/OS for evaluation using Universal Installer
 ---
 
 This page explains about Terraform and the evaluation methods that are supported for both DC/OS OSS (default) and DC/OS Enterprise installation. You can evaluate the installation of DC/OS cluster based on your requirements.
```
"This page explains about the Universal Installer..."

yishan-lin commented on pages/1.12/installing/evaluation/index.md:
```diff
@@ -3,22 +3,22 @@ layout: layout.pug
 - Remove the confusion around which DC/OS installation method should be used in any given scenario. This automated tool helps to build modules that codify best practices for each stage in the cluster life-cycle and hook necessary modules into an existing infrastructure. 
 	
-Terraform is the recommended method for provisioning, deploying, installing, and upgrading DC/OS on a subset of our supported platforms.
+Universal Installer is the recommended tool for provisioning, deploying, installing, and upgrading DC/OS on a subset of our supported platforms.
```
The Universal Installer is the recommended tool...

yishan-lin commented on pages/1.12/installing/index.md:
Shouldn't this be "Mesosphere Supported Installation"?  no hyphens, capitalize

anithamuthu79 replied:
This is “Mesosphere-supported” and “Community-supported” (as they are compound adjectives).

yishan-lin commented on pages/1.12/installing/index.md:
Same things here, "Community Supported Installation"
